data/transition-sites/phe_datag.yml

```diff
@@ -0,0 +1,7 @@
+---
+site: phe_datag
+whitehall_slug: public-health-england
+homepage: https://www.gov.uk/government/organisations/public-health-england
+tna_timestamp: 20150505172621
```

Comment:
That was an idle comment - more of a note - anyway, not something you needed to change. :snail: